README.md

```diff
@@ -11,6 +11,27 @@ The Mixpanel library for iOS is an open source project, and we'd love to see you
 Mixpanel supports `Cocoapods` for easy installation.
 To Install, see our **[full documentation »](https://mixpanel.com/help/reference/ios)**
 
+If you choose to go with this method and want to activate the logging and debugging modes in the SDK
+(see "Debugging and logging" in **[the documentation »](https://mixpanel.com/help/reference/ios)**), you can use a
+`post_install` hook in your Podfile, like so:
+
+```ruby
+post_install do |installer|
+    installer.pods_project.targets.each do |target|
+        target.build_configurations.each do |config|
+            s = config.build_settings['GCC_PREPROCESSOR_DEFINITIONS']
+            if s==nil then s = [ '$(inherited)' ] end
+
+            if target.name == "Pods-MyProject-Mixpanel"
+                s.push('MIXPANEL_DEBUG=1')
+                s.push('MIXPANEL_ERROR=1')
+            end
+            config.build_settings['GCC_PREPROCESSOR_DEFINITIONS'] = s
+        end
+    end
+end
```

Comment:
I trust you on this :stuck_out_tongue: 
What about @kylef's `s ||= [ '$(inherited)' ]`, that looks nice!